plugin.cfg

```diff
@@ -61,6 +61,7 @@ secondary:secondary
 etcd:etcd
 loop:loop
 forward:forward
+forwardcrd:forwardcrd
```

Comment:
_forwardcrd_ should come before _forward_. Otherwise, a default upstream forwarder, such as the one in the default K8s file `forward . /etc/resolv.conf`, would prevent _forwardcrd_ in the same server block from serving any queries.